lib/lita-lean_tc.rb

```diff
@@ -120,7 +120,7 @@ def stream(response)
 
       def start_review_timer
         every_with_logged_errors(TIMER_INTERVAL) do |timer|
-          persistent_every("review-column-activity", days_in_seconds(1)) do
+          daily_at("23:00", [:sunday, :monday, :tuesday, :wednesday, :thursday], "review-column-activity") do
```

Comment:
I think the `daily_at` method will need to be defined - there's a good example in `lib/hello_slack.rb`